cmd/download.go

```diff
@@ -109,10 +114,18 @@ func templatizeImageset(release, folder string, aiImages, additionalImages []str
 	channel := r[0] + "." + r[1]
 	version := release
 
+	// Validate hub version is something like x.y.z
```

Comment:
Cobra supports a PersistentPreRunE function (https://github.com/spf13/cobra/blob/main/user_guide.md#prerun-and-postrun-hooks) that looks like it could be a good place for adding validation of parameter values, like checking patterns. Maybe a cleanup item we could do at some point, to move any existing parameter validation into a PersistentPreRunE